JitsiConference.js

```diff
@@ -1069,6 +1185,11 @@ JitsiConference.prototype.onRemoteTrackRemoved = function(removedTrack) {
     }, this);
 
     if (!consumed) {
+        if ((this.p2pEstablished && !removedTrack.isP2P)
+             || (!this.p2pEstablished && removedTrack.isP2P)) {
+            // This track has been removed explicitly by P2PEnabledConference
```

Comment:
The thing is that it can be either removed by the P2P logic or as a result of 'source-remove'. The P2P removes JVB remote tracks when in P2P and vice versa. But when it's a result of 'source-remove' it is an error which should be logged. Does it make sense ? If this explanation is enough I will add it to the comment.